packages/react/src/select/popup/SelectPopup.tsx

```diff
@@ -317,7 +318,7 @@ export const SelectPopup = React.forwardRef(function SelectPopup(
         return;
       }
 
-      if (reachedMaxHeightRef.current || !alignItemWithTriggerActive) {
+      if (reachedMaxHeightRef.current) {
```

Comment:
Not relevant to the bug, but there's already an `!alignItemWithTriggerActive` check above so this is needless.